lib/ar2dto/config.rb

```diff
@@ -1,23 +1,23 @@
 # frozen_string_literal: true
 
-require "singleton"
-
 module AR2DTO
   class Config
     include Singleton
 
     def self.reset!
       instance.active_model_compliance = true
       instance.except = []
-      instance.replace_suffix = { from: "", to: "DTO" }
+      instance.delete_suffix = ""
+      instance.add_suffix = "DTO"
```

Comment:
the replace_suffix option is complicated, e.g. what if someone used it with a string instead of a hash?